front/app/containers/Admin/index.tsx

```diff
@@ -96,8 +96,10 @@ const AdminPage = memo<Props & WithRouterProps>(
     const [adminFullWidth, setAdminFullWidth] = useState(false);
     const [adminNoPadding, setAdminNoPadding] = useState(false);
 
-    const userCanViewAdmin = usePermission({
-      item: { type: 'route', path: '/admin' },
```

Comment:
The problem was that '/admin' is one of the moderator routes and it gets checked to be a substring of item.path, which is '/admin' here. So it returns true for each admin route in isModeratorRoute (see below).